po/R-zh_CN.po

```diff
@@ -158,7 +152,7 @@ msgstr ""
 msgid ""
 "'between' arguments are all POSIXct but have mismatched tzone attributes: "
 "%s. The UTC times will be compared."
-msgstr "'between' 的参数均为 POSIXct 类型但时区属性（tzone）不匹配："
+msgstr "<fix fuzzy translation>'between' 的参数均为 POSIXct 类型但时区属性（tzone）不匹配："
```

Comment:
```suggestion
msgstr "<fix fuzzy translation>'between' 的参数均为 POSIXct 类型但时区属性（tzone）不匹配：%s。用 UTC 比较。"
```

@hongyuanjia I've added markers for all the pieces that need addressing, either with `<fix fuzzy translation>` as here or `<needs translation>` as e.g. L187.

My hope is we can use the `suggestion` feature on GitHub (as done here in this comment) to make it easier to finish the translations directly in the PR.

LMK if that makes sense to you, and if my sense that this is easier is right/wrong.

Reply:
I got it. I have already finished all `#, fuzzy` ones and added translations for those messages that are currently missing. I will be sure to fetch your updates to see if any more updates are needed.

I use GitHub Copilot to do the essential translations and check manually to ensure it makes sense and feels more natural. Currently, it did a really good job ;)

Reply:
Indeed, there are quite a lot conflicts... I will open a new PR. Please see #5875.